.github/workflows/docker_master.yaml

```diff
@@ -3,41 +3,13 @@ name: Docker Master Release
 on:
   push:
     branches: [ master ]
-  pull_request:
-    branches: [ master ]
```

Comment:
[the `pull_request` event is fired when a pull request occurs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

This has [never ran before](https://github.com/CollinHeist/TitleCardMaker/actions?query=event%3Apull_request+branch%3Amaster)